Sources/OpenAIKit/Configuration.swift

```diff
@@ -1,26 +1,33 @@
 import NIOHTTP1
+import Foundation
 
 public struct Configuration {
     public let apiKey: String
     public let organization: String?
+    public let host: String
+    var headers: HTTPHeaders
```

Comment:
this can be a let OR you don't need the local `headers` property in the init. Right?

Reply:
We do need `headers` here. I already updated it to let